src/utils/db/db.ts

```diff
@@ -496,6 +496,44 @@ export class DB {
         return person
     }
 
+    public async setPersonAsIdentified(person: Person): Promise<void> {
+        await this.postgresTransaction(async (client) => {
```

Comment:
Not sure if this is the right tradeoff here:
problems:
* we're more likely to now not write to Postgres (e.g. Kafka is down, or there was a problem with the write), which we treat as source of truth for people stuff & where we can recover stuff from (because everything is async user called identify and assumed things are good, but nothing happened is more likely now)
* number of active transactions increases (perf decrease)

How immediately do Postgres & ClickHouse need to be in sync here? An alternative to consider could be to have some cron job after some delay (significantly bigger than Kafka normal delay), that updates the identified for CH (in case we failed here). It's quite easy as we only ever set it to true (so we know exactly when it needs to happen & don't need to worry about anyone overriding the value to something else).

Reply:
Honestly, I don't think this is that important to warrant a cron job.

The chances of us failing in between those lines is minimal and very low-impact. So I'd just advocate for reverting the transaction change and leaving as is.

Reply:
sound good to me.

Reply:
After thinking for a while I went back to prioritizing integrity (even with low probability scenarios) over performance here.

We do a check not long before this runs to see if the person was already identified, so we also shouldn't be doubling too many messages to Kafka. Doesn't hurt to be cautious - thanks for pushing here.